hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -154,6 +155,12 @@ public class HoodieTableConfig extends HoodieConfig {
       .withDocumentation("Payload class to use for performing compactions, i.e merge delta logs with current base file and then "
           + " produce a new base file.");
 
+  public static final ConfigProperty<String> MERGE_CLASS_NAME = ConfigProperty
+      .key("hoodie.compaction.merge.class")
+      .defaultValue(HoodieAvroRecordMerge.class.getName())
+      .withDocumentation("Merge class provide stateless component interface for merging records, and support various HoodieRecord "
```

Comment:
I'm -1 for a separate merge clazz that is only for merging, can we elaborate a little more for just reusing the merge clazz for write path ?

Reply:
I'm also confused about config such as write config and compact config. At present, I write it after imitating payload class. Wait for unified reconfiguration later?